install/kubernetes/cilium/templates/hubble-ui-deployment.yaml

```diff
@@ -63,6 +63,16 @@ spec:
             - name: FLOWS_API_ADDR
```

Comment:
nit: I think this would be easier to read if you duplicated that line here, and move it into the `if` block, i.e.

```
{{- if .Values.hubble.relay.tls.server.enabled }}
            - name: FLOWS_API_ADDR
              value: "hubble-relay:443"
            - ...
{{- else }}
            - name: FLOWS_API_ADDR
              value: "hubble-relay:80"
{{- end }}
```

Reply:
Done